src/components/views/elements/EditableItemList.js

```diff
@@ -84,7 +84,9 @@ module.exports = React.createClass({
         onNewItemChanged: PropTypes.func,
         onItemAdded: PropTypes.func,
         onItemEdited: PropTypes.func,
-        onItemRemoved: PropTypes. func,
```

Comment:
No idea how this is even possible but empirically it's not a syntax error. Babel might be dealing with it?

Reply:
heh, I can well believe that the dot operator accepts whitespace around it